runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java

```diff
@@ -119,6 +120,20 @@ private void waitForWorkflow(@NonNull UUID workflow, @NonNull ManagementClient c
         throw new TimeoutException();
     }
 
+    /*
+     * private method to handle exceptions while sendRequest()
+     */
+    private void handleException(RuntimeException runtimeException) {
+        final Throwable cause = ExceptionUtils.getRootCause(runtimeException);
+        if (cause instanceof TimeoutException ||
```

Comment:
TimeoutException can't appear here, because it already handled in `invoke` method